apps/src/code-studio/components/ShareDialog.jsx

```diff
@@ -37,26 +47,137 @@ var ShareDialog = React.createClass({
     this.setState({isOpen: false});
   },
 
+  showSendToPhone: function (event) {
```

Comment:
True of functions throughout this file. Some have the function keyword, others do not.